webpack.config.js

```diff
@@ -2,6 +2,15 @@ const HtmlWebPackPlugin = require("html-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
 const path = require("path");
 
+const webpack = require('webpack');
+const dotenv = require('dotenv');
+
+const env = dotenv.config({path: '.env.local'}).parsed;
+const envKeys = Object.keys(env).reduce((prev, next) => {
+  prev[`process.env.${next}`] = JSON.stringify(env[next]);
+  return prev;
+}, {});
```

Comment:
I was wondering if we can remove these steps that handle specific `.env` files and I had a look into the webpack documentation. They recommend the usage of [dotenv-webpack](https://webpack.js.org/plugins/environment-plugin/#dotenvplugin). Do you think that we could use it?

Reply:
Nice suggestion. I've changed the library and now it looks better, thanks.

Reply:
Thanks!!!!